docs/en/enterprise-edition/policy-reference/aws-policies/aws-iam-policies/ensure-the-aws-execution-role-arn-and-task-role-arn-are-different-in-ecs-task-definitions.adoc

```diff
@@ -23,8 +23,32 @@
 |=== 
 
 
+=== Description 
 
-=== Description
+In Amazon ECS, the execution role and the task role serve different purposes and should be distinct:
 
+* The *execution role* is used by the ECS agent to make AWS API calls on your behalf.
+* The *task role* grants permissions to the tasks themselves.
 
-=== Fix - Buidtime
+Using the same role for both can lead to unnecessary permissions being granted, increasing security risks. Separating these roles ensures a more granular and secure permission model.
```

Comment:
Although it's fine, consider an explanation such as: If the same role is used for both purposes, it might lead to a broader set of permissions than required. For example, the execution role might end up with unnecessary task-specific permissions, or the task role might have permissions that are not relevant to the execution role.